.ci/openjdk-projects-to-test-on.config

```diff
@@ -0,0 +1,8 @@
+# List of GIT repositories to clone / pull for checking with Checkstyle
+# File format: REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+# Please note that bash comments works in this file
+#
+# This file has been created to alleviate confusion in the contribution
+# repository's projects-to-test-on.properties file; these projects
+# must be used with special configurations.
+openjdk14|git|https://github.com/openjdk/jdk14|master||
```

Comment:
Why is this file needed? Why can't we use contribution's file and tell it to target openjdk14 only?

Reply:
It is very special line, it  is not usable for regular groovy files.
I even removed openjfk from contribution repo, as it caused too much confusion to contributors.

Reply:
I am resolving this. I thought the line was still in contribution.